src/libs/auth.ts

```diff
@@ -399,40 +415,27 @@ export const isAzureUser = (): boolean => {
   return _.startsWith('https://login.microsoftonline.com', getUser().idp!);
 };
 
-export const processUser = (user: OidcUser | null, isSignInEvent: boolean): void => {
+export const doUserLoaded = (user: OidcUser): void => {
```

Comment:
I ended up going with `userLoadedEventHandler` sort of following the [.NET guidelines] (https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-events) for naming convention. Although this may not be the style we want to adopt here.